GliaWidgets/Sources/ViewController/Chat/ChatViewController.swift

```diff
@@ -54,114 +54,116 @@ class ChatViewController: EngagementViewController, MediaUpgradePresenter,
             case .setMessageEntryEnabled(let enabled):
-                view.messageEntryView.isEnabled = enabled
+                view?.messageEntryView.isEnabled = enabled
             case .setChoiceCardInputModeEnabled(let enabled):
-                view.messageEntryView.isChoiceCardModeEnabled = enabled
+                view?.messageEntryView.isChoiceCardModeEnabled = enabled
             case .setMessageText(let text):
-                view.messageEntryView.messageText = text
+                view?.messageEntryView.messageText = text
             case .sendButtonHidden(let hidden):
-                view.messageEntryView.showsSendButton = !hidden
+                view?.messageEntryView.showsSendButton = !hidden
             case .pickMediaButtonEnabled(let enabled):
-                view.messageEntryView.pickMediaButton.isEnabled = enabled
+                view?.messageEntryView.pickMediaButton.isEnabled = enabled
             case .appendRows(let count, let section, let animated):
-                view.appendRows(count, to: section, animated: animated)
+                view?.appendRows(count, to: section, animated: animated)
             case .refreshRow(let row, let section, let animated):
-                view.refreshRow(row, in: section, animated: animated)
+                view?.refreshRow(row, in: section, animated: animated)
             case .refreshRows(let rows, let section, let animated):
-                view.refreshRows(rows, in: section, animated: animated)
+                view?.refreshRows(rows, in: section, animated: animated)
             case let .refreshSection(section, animated):
-                view.refreshSection(section, animated: animated)
+                view?.refreshSection(section, animated: animated)
             case .refreshAll:
-                view.refreshAll()
+                view?.refreshAll()
             case .scrollToBottom(let animated):
-                view.scrollToBottom(animated: animated)
+                view?.scrollToBottom(animated: animated)
             case .updateItemsUserImage(let animated):
-                view.updateItemsUserImage(animated: animated)
+                view?.updateItemsUserImage(animated: animated)
             case .addUpload:
                 // Handled by data-driven SecureConversations.FileUploadListView.
                 break
             case .removeUpload:
                 // Handled by data-driven SecureConversations.FileUploadListView.
                 break
             case .removeAllUploads:
-                view.messageEntryView.uploadListView.removeAllUploadViews()
+                view?.messageEntryView.uploadListView.removeAllUploadViews()
             case .presentMediaPicker(let itemSelected):
+                guard let view = view else { return }
                 self?.presentMediaPicker(
                     from: view.messageEntryView.pickMediaButton,
                     itemSelected: itemSelected
                 )
+                break
```

Comment:
`break` can be removed

Reply:
fixed [8e5ef50](https://github.com/salemove/ios-sdk-widgets/pull/573/commits/8e5ef50e46d3e5f46eb24dab70b3fff962ed3071)